### ISSUE TITLE
Missing img/i-f7-material.png and i-f7-ios.png causes webpack build to fail.  Should include these in dist/umd/ in your repo?

### ISSUE THREAD

**Issue by uglycoyote** [bencompton/framework7-react#87]
I'm adding framework7-react to an existing webpack build.    (i have an existing project so it did not make sense to start from scratch using create-react-app)

I used npm to install using ```npm install framework7-react --save``` as mentioned under the **Installing in an App with a Build Pipeline** section of your documentation.

My process for adding to my existing webpack build was

1. Importing into my Typescript entry point:

```
import * as Framework7 from 'framework7-react'
import { Framework7App, Statusbar, Views, View, Pages, Page, Toolbar, Link, Navbar } from "framework7-react";
```

At the moment this file just does a ```ReactDOM.render``` of the component copied from your website's App Layout page:  https://framework7.io/react/app-layout.html

2. Importing the relevant CSS files into my own LESS file

```
@import "../node_modules/framework7-react/dist/umd/css/framework7.material.colors.css";
@import "../node_modules/framework7-react/dist/umd/css/framework7.material.css";
@import "../node_modules/framework7-react/dist/umd/css/framework7.ios.colors.css";
@import "../node_modules/framework7-react/dist/umd/css/framework7.ios.css";
```

By doing so, webpack recognizes the framework7-react sources and CSS as needing to be bundled.

Following this procedure, webpack gives errors about missing image files:

```
    ERROR in ./node_modules/css-loader!./node_modules/framework7-react/dist/umd/css/framework7.material.css
    Module not found: Error: Can't resolve '../img/i-f7-material.png' in '/Users/mike/development/web/SonicPines/client/node_modules/framework7-react/dist/umd/css'
     @ ./node_modules/css-loader!./node_modules/framework7-react/dist/umd/css/framework7.material.css 6:60158-60193
     @ ./node_modules/css-loader!./node_modules/less-loader/dist/cjs.js!./styles/Framework7Styles.less
     @ ./styles/Framework7Styles.less
```

and if I try to run my App in Chrome, I get a corresponding runtime error:

```
Uncaught Error: Cannot find module "../img/i-f7-material.png"
    at webpackMissingModule (framework7.material.css:6)
    at Object.278 (framework7.material.css:6)
    at __webpack_require__ (bootstrap 05fd7dc9ed98e4fbcd80:19)
    at Object.276 (Framework7Styles.less:4)
    at __webpack_require__ (bootstrap 05fd7dc9ed98e4fbcd80:19)
    at Object.275 (Framework7Styles.less?d1ef:4)
    at __webpack_require__ (bootstrap 05fd7dc9ed98e4fbcd80:19)
    at 21.module.exports (bootstrap 05fd7dc9ed98e4fbcd80:62)
    at bootstrap 05fd7dc9ed98e4fbcd80:62
```
which is fatal (the app renders with Times Roman so clearly the failure to bundle/load this image has prevented any of the Framework7 CSS rules from functioning at all)

**Workaround**

I was able to work around this issue by manually copying the required images (there are only two of them) from a previously-created sandbox app that I created elsewhere using  `phonegap create mobile --template framework7`.    I manually copied them to `node_modules/framework7-react/dist/umd/img`.

```
➜  umd git:(master) ✗ mkdir img
➜  umd git:(master) ✗ cd img
➜  img git:(master) ✗ cp ../../../../../../mobile/www/lib/framework7/img/* .
➜  img git:(master) ✗ ls
i-f7-ios.png      i-f7-material.png
```
**Solution**

I think either of the following would be an acceptable solution

1. This repo should include the directory `dist/umd/img` with these two images, or
2.  The CSS rules which mention the images should be removed

The latter solution may be unacceptable to you since I imagine you are getting these CSS files directly from the main Framework7 repo and you don't want to have unique versions 

**Note**

This may be an issue that would go away if this repo was brought up to date with the latest stuff in the main Framework 7 repo.  I investigated on the main Framework7 repo,  It appears that the newer versions of the CSS don't include any .png's 

e.g. [these CSS from the 1.6.5 release look like yours](https://github.com/framework7io/framework7/blob/d575d3972954c74a8e3d58e969fbc910dcc790ff/dist/css/framework7.material.css) however the [latest version of the CSS](https://github.com/framework7io/framework7/blob/master/dist/css/framework7.md.css) does not mention any .png's.  Also the CSS files appear to have been renamed since the start of 2.0

**Comment by bencompton** [comment]
You might try NPM installing Framework7 in addition to Framework7 React, and then importing the CSS from Framework7. That's what the F7 React CRA template [does](https://github.com/bencompton/framework7-react-app-template/blob/master/src/index.js).

**Comment by uglycoyote** [comment]
Thanks.   I will try that.   If that's the recommended procedure, it might be good to specify this in the installation document.   Since the "Installing in an App with a Build Pipeline" section said to install only React and framework7-react, I assumed that also installing Framework7 would be redundant and possibly cause compatibility issues due to having conflicting parts.  Especially now that I have discovered that framework7-react is not up to date with the 2.0 version of Framework7, I'm a bit surprised that you say they can work together.   But now that you have suggested it I will give that a try.

**Comment by bencompton** [comment]
Yeah, I agree that it isn't a very refined solution to install Framework7 alongside F7 React just to get the CSS. The kitchen sink has the static CSS files manually copied from F7 in a "css" directory, which also isn't the best solution. I'm going to keep this issue open, since this is something that could definitely be improved.

In the meantime, installing F7 beside F7 React shouldn't create any conflicts, since F7 React uses its own bundled version of F7. Just make sure it is 1.6.5 you install instead of 2.x.
